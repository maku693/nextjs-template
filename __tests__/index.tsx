import { render } from "@testing-library/react";
import Index from "@/pages/index";

describe("Index", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<Index />);
    expect(asFragment()).toMatchSnapshot();
  });
});
