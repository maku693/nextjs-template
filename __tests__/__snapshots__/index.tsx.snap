// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should match snapshot 1`] = `
<DocumentFragment>
  <div>
    Hello, Next.js
  </div>
</DocumentFragment>
`;
